### DISCUSSION TITLE
Using `game.clock_increment`

### DISCUSSION THREAD

**Discussion by TheYoBots** [lichess-bot-devs/lichess-bot#828] [Q&A] [answered]
Will using `game.clock_increment == 0` cause any issue in a function in the `lichess-bot.py` or `engine_wrapper.py` file which has `game: model.Game` as one of its parameters?

**Comment by TheYoBots** [comment, marked as answer]
It works when I convert `0` to timedelta:
`if game.clock_increment == seconds(0)`

**Reply by MarkZH** [reply]
Right. The member `Game.clock_increment` is created on [this line](https://github.com/lichess-bot-devs/lichess-bot/blob/2144f78545fb8fa0f8f78f248c3d0d0bd0f6a40d/model.py#L155) as a `datetime.timedelta` type.
